modules/ROOT/pages/dw-multipart-types.adoc

```diff
@@ -5,12 +5,13 @@
 
 | Multipart
 | `type Multipart = { preamble?: String, parts: { _?: MultipartPart } }`
-| `MultiPart` type, a data structure for a complete `Multipart` format. See the output example for the Multipart `form` function xref:dw-multipart-functions-form.adoc[documentation].
+| `MultiPart` type, a data structure for a complete `Multipart` format. See the
```

Comment:
Need to add the link to xref:dw-multipart-functions-form.adoc[documentation] as an